app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -0,0 +1,120 @@
+import { BoxProps } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Autocomplete } from "@material-ui/lab";
+import React, { useEffect, useRef, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import Button from "../../../components/Button";
+import TextField from "../../../components/TextField";
+import { User } from "../../../pb/api_pb";
+import { service } from "../../../service";
+import { useAppDispatch, useTypedSelector } from "../../../store";
+import { fetchUsers, getUsers } from "../../userCache";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface GroupChatListProps extends BoxProps {
+  createGroupChat: (title: string, users: User.AsObject[]) => void;
+}
+
+export function useFindUsers(ids: number[]): User.AsObject[] {
+  const notFound: number[] = [];
+  const usersRef = useRef<User.AsObject[]>([]);
+  const [idsString, setIdsString] = useState<string>("");
+  const allUsers = useTypedSelector((state) => getUsers(state));
+  const dispatch = useAppDispatch();
+  usersRef.current = [];
+  const users = usersRef.current;
+  for (const id of ids) {
+    const user = allUsers[id];
+    if (user) {
+      users.push(user.user);
+    } else {
+      notFound.push(id);
+    }
+  }
+  const newIdsString = "" + ids.sort((a, b) => a - b);
+  if (newIdsString !== idsString) {
+    setIdsString(newIdsString);
+    if (notFound.length) {
+      dispatch(fetchUsers({ userIds: notFound }));
+    }
+  }
+  return usersRef.current;
+}
+
+interface CreateGroupChatFormData {
+  title: string;
+  users: User.AsObject[];
+}
+
+export default function CreateGroupChat({
+  createGroupChat,
+}: GroupChatListProps) {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [friendIds, setFriendIds] = React.useState<number[]>([]);
+  const friends = useFindUsers(friendIds);
+
+  useEffect(() => {
+    (async () => {
+      setLoading(true);
+      try {
+        setFriendIds(await service.api.listFriends());
+      } catch (error) {
+        setError(error.message);
+      }
+      setLoading(false);
+    })();
+  }, []);
+  const { control, register, handleSubmit } = useForm<
+    CreateGroupChatFormData
+  >();
+
+  const classes = useStyles();
+
+  const onSubmit = handleSubmit(async (data: CreateGroupChatFormData) => {
+    try {
+      createGroupChat(data.title, data.users);
+    } catch (error) {
+      setError(error.message);
+    }
+  });
+  return (
+    <form onSubmit={onSubmit}>
+      <TextField label="Title" name="title" inputRef={register} />
+      <Controller
+        control={control}
+        defaultValue={friends}
+        name="users"
+        onChange={([, data]: any) => data}
+        render={({ onChange }) => (
+          <Autocomplete<User.AsObject>
+            onChange={(_, value) => {
+              onChange(value);
+            }}
+            multiple={true as any}
```

Comment:
```suggestion
            multiple={true}
```
Unnecessary casting?

Reply:
It is a type assertion, not a type cast. When people use `as any` they typically have trouble solving a type complaint from Typescript. So did I. But I will try again. For some reason, which I did not understand, true was not assignable.

Reply:
The type on this is `boolean | undefined` so you shouldn't need this hopefully. I think if you need to do something like this, it's still better to use `unknown` instead, a slightly stricter version of `any` which is recommended by TypeScript guide themselves - https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any

Reply:
It was caused by using the generic type. It sets this property type to undefined.